addons/calendar/models/calendar_recurrence.py

```diff
@@ -216,6 +216,8 @@ def _apply_recurrence(self, specific_values_creation=None, no_send_edit=False):
             events_to_keep, ranges = recurrence._reconcile_events(ranges)
             keep |= events_to_keep
             [base_values] = event.copy_data()
+            attendee_ids = [dict(state=rec['state'], partner_id=rec['partner_id'].id) for rec in event['attendee_ids']]
```

Comment:
`event` is the base event of the recurrence and it may not be new when `_apply_recurrence` is called.
That means, attendees of this base event may already have answered to this event. 
As you use base event attendees statuses to fill `base_values['attendee_ids']`, that means you may set new attendees statuses to something different from "need action".
To avoid that, I would hardcode the "need action" status instead of using the one from the base event. What do you think ?

Reply:
Thanks for your comment.
It would solve the problem of sending emails to the attendees. But it could overwrite accepted and declined status.

Reply:
See my comment. When you convert an existing event, you already get 'accepted' for self and need action for the others.